plugins/plugin-chart-echarts/src/utils/series.ts

```diff
@@ -200,3 +200,7 @@ export function dedupSeries(series: SeriesOption[]): SeriesOption[] {
     };
   });
 }
+
+export function sanitizeHtml(text: string): string {
+  return format.encodeHTML(text);
+}
```

Comment:
@stephenLYZ I left the original `sanitizeHtml` function in place so we can more easily swap `encodeHTML` out if needed later.